### DISCUSSION TITLE
Unable to export data for multiple skeletons in the same frame

### DISCUSSION THREAD

**Discussion by Jaspermoray** [talmolab/sleap#1543] [Help!]
Hi,

I am trying to export the data for multiple instances in one frame, but it only seems to export one instance per frame. I tried first using the .analysis.h5 filetype, and it only stored one instance per frame. Then I tried just exporting a CSV file from the GUI, and that only was able to store one animal per frame as well. Am I missing an option to export multiple instances per frame? 

Best,
Jasper

**Comment by roomrys** [comment]
Hi @Jaspermoray,

The analysis file uses the instance's track to organize the pose data. If you have no tracks in your project, then the export function assumes your project is a single instance project and exports just one instance per frame. 

Do you have tracks assigned to your instances? If you don't, then they need to be assigned, but also, I am wondering if (and why) it would be helpful to you to export pose estimations without having them sorted into tracks/identities (for development purposes)?

Thanks,
Liezl
